packages/account/src/Components/personal-details/__tests__/personal-details.spec.js

```diff
@@ -61,17 +61,25 @@ const runCommonFormfieldsTests = is_svg => {
     expect(screen.queryByTestId('tax_residence')).toBeInTheDocument();
     expect(screen.queryByTestId('tax_residence_mobile')).not.toBeInTheDocument();
 
-    expect(
-        screen.getByText(/Please enter your first name as in your official identity documents./i)
-    ).toBeInTheDocument();
+    if (is_svg) {
+        expect(screen.getByText(/your first name as in your identity document/i)).toBeInTheDocument();
```

Comment:
Why we need to add condition check in test cases?

Reply:
one thing is to remove these check and rewrite the according to the test requirement as this text `Please enter your first name as in your official identity documents.` is only applicable for mf accounts and this was there to be tested for cr clients as well